.tekton/infra-deployment-push.yaml

```diff
@@ -40,3 +44,20 @@ spec:
         taskRef:
           bundle: quay.io/redhat-appstudio-tekton-catalog/task-update-infra-deployments:0.1
           name: update-infra-deployments
+    finally:
+      - name: slack-webhook-notification
+        when:
+          - input: $(params.slack-webhook-notification-team)
+            operator: notin
+            values: [""]
+          - input: $(tasks.status)
+            operator: in
+            values: ["Failed"]
+        params:
+          - name: message
+            value: Tekton pipelineRun $(context.pipelineRun.name) failed
```

Comment:
Can we link it to the URL of the PipelineRun (or add the link in the message)? So that we can directly click and go to the referred run.

Reply:
This change needs to be done upstream in the task itself https://github.com/redhat-appstudio/build-definitions/blob/main/task/slack-webhook-notification/0.1/slack-webhook-notification.yaml